taskfile/ast/taskfile.go

```diff
@@ -55,8 +55,8 @@ func (t1 *Taskfile) Merge(t2 *Taskfile, include *Include) error {
 	}
 	t1.Vars.Merge(t2.Vars, include)
 	t1.Env.Merge(t2.Env, include)
-	t1.Tasks.Merge(t2.Tasks, include, t1.Vars)
-	return nil
+	err := t1.Tasks.Merge(t2.Tasks, include, t1.Vars)
```

Comment:
If you use only for returning it. So, without wrapping value on error, then you could simply


```suggestion
	return t1.Tasks.Merge(t2.Tasks, include, t1.Vars)
```